vng_api_common/notifications/handlers.py

```diff
@@ -8,4 +17,44 @@ def handle(self, message: dict) -> None:
         logger.info("Received notification %r", message)
 
 
+class AuthHandler:
+    def handle_auth(self, message: dict) -> None:
+
+        # uuids in AC DB are synchronized with uuids in local DB
+        uuid = get_identifier_from_path(message['resource_url'])
+        
+        if message['actie'] == 'delete':
+            Applicatie.objects.get(uuid=uuid).delete()
```

Comment:
will crash if the object didn't exist locally (yet), which is possible if you've missed a notification.

```suggestion
            Applicatie.objects.filter(uuid=uuid).delete()
```
is a bit more robust